components/lsp-form/LongShort.tsx

```diff
@@ -31,22 +37,34 @@ const LongShort: FC<Props> = ({
           placeholder="0.0"
           value={longTokenAmount}
           setValue={setLongTokenAmount}
+          additionalEffects={(e) => {
+            setShortTokenAmount(e.target.value);
+            const newAmount =
+              Number(e.target.value) * Number(collateralPerPair);
+            setAmount(newAmount.toString());
+          }}
         />
         <TextInput
           label="short token"
           labelPlacement={"overlap" as LabelPlacement}
           placeholder="0.0"
           value={shortTokenAmount}
           setValue={setShortTokenAmount}
+          additionalEffects={(e) => {
+            setLongTokenAmount(e.target.value);
+            const newAmount =
+              Number(e.target.value) * Number(collateralPerPair);
+            setAmount(newAmount.toString());
```

Comment:
Agreed. Can we have some function that we call that's able to detect which box is the "selected" one so we avoid updating that one? Otherwise, there may be odd situations due to rounding, etc.

Reply:
Makes sense, done.